Sources/SwiftDocC/Model/Identifier.swift

```diff
@@ -207,31 +212,39 @@ public struct ResolvedTopicReference: Hashable, Codable, Equatable, CustomString
     }
     
     /// - Note: The `path` parameter is escaped to a path readable string.
-    public init(bundleIdentifier: String, path: String, fragment: String? = nil, sourceLanguage: SourceLanguage) {
-        self.init(bundleIdentifier: bundleIdentifier, path: path, fragment: fragment, sourceLanguages: [sourceLanguage])
+    public init(id: DocumentationBundle.Identifier, path: String, fragment: String? = nil, sourceLanguage: SourceLanguage) {
+        self.init(id: id, path: path, fragment: fragment, sourceLanguages: [sourceLanguage])
```

Comment:
Yes, it's possible to keep the parameter names the same (the "function signature" is the parameter and return types). 


I do have (separate) plans to deprecate the "documentation bundle" term because it's confusing. It's often used to mean "documentation catalog" but they are not the same, neither in structure or purpose. 

I remove the phase "bundle" in this new public API so that I wouldn't need to introduce and deprecate it in the same release but perhaps I'm getting ahead of myself and it would be better to make that change when we deprecate the term "bundle".

Reply:
Updated in https://github.com/swiftlang/swift-docc/pull/1069/commits/f97e6d39751715fcac3bee38caab181c60e6ac3e